docs/licenses/python.txt

```diff
@@ -1,4 +1,4 @@
-https://docs.python.org/3.12/license.html
+https://docs.python.org/3.13/license.html
 
 The MIT License
```

Comment:
Python does not use the MIT license. This is the Locust license, which is duplicated two times. Please work with @dsotirho-ucsc to have him create a ticket.

Reply:
Daniel created https://github.com/DataBiosphere/azul/issues/7372.